files/nrpe/check_openstack.py

```diff
@@ -201,6 +201,16 @@ def keystone_session_v3(options):
     sessionx = session.Session(auth=auth)
     return sessionx
 
+def get_project_id(session, name):
+    keystone = keystoneclientv3.Client(session=session)
+    projects = keystone.projects.list()
+    project_list = list(filter(lambda d: name == d.name , projects))
+    if len(project_list) != 1:
+        print("there should only be on project")
```

Comment:
Throughout check_openstack.py, so far we don't do any exception reporting with `print`. IMO this should be a `logging.warn` or more severe. Otherwise the PR looks good and I am going to approve it already.

Reply:
updating.